benchmarks/benchmarks/spatial.py

```diff
@@ -290,6 +290,29 @@ def time_count_neighbors_shallow(self, mn1n2, Nr):
         """
         self.T1s.count_neighbors(self.T2s, self.r)
 
+
+class Remove(Benchmark):
+    params = [
+        [
+          (2, 1000),
+          (8, 1000),
+          (16, 1000)
```

Comment:
1000 points is quite small for a kdtree, other benchmarks use 10 thousand points and I would go even higher if the benchmarking time is still reasonable.